07.bowling_object/frame.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require_relative 'shot'
+
+class Frame
+  def initialize(first_mark, second_mark = nil, third_mark = nil)
+    @first_shot = Shot.new(first_mark)
+    @second_shot = Shot.new(second_mark)
+    @third_shot = Shot.new(third_mark)
```

Comment:
```suggestion
  def initialize(*marks)
    @shots = marks.map { |mark| Shot.new(mark) }
```

のようにして、shotを配列として保持すると意外と実装がスッキリするかも？